doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,118 @@
+
+The raw packet transmit header elements and their description is provided below:
+
+.. list-table:: Wi-Fi raw packet transmission header elements
+   :header-rows: 1
+
+   * - Transmit and PHY parameters
+     - Description
+   * - magic_num
+     - magic number to identify a raw packet. It is set to 0x12345678.
+   * - data_rate
+     - Data rate at which packet is to be transmitted. Depends on ``tx_mode`` parameter. If ``tx_mode`` is set to legacy mode, data rate is the number provided. If ``tx_mode`` is set to HT/VHT/HE mode, the data rate is the MCS rate.
+   * - packet_length
+     - Packet length of 802.11 raw packet excluding the raw transmit packet header length.
+   * - tx_mode
+     - Mode describing if packet is VHT, HT, HE or Legacy.
+   * - queue
+     - Wi-Fi access category mapping for packet.
+   * - raw_tx_flag
+     - Flag indicating raw packet transmission. This is reserved for driver use.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/raw_tx_pkt_header`` for the raw packet header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_ac`` for setting the ``queue`` parameter in the raw packet transmit header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_rawtx_mode`` for setting the ``tx_mode``  parameter in the raw packet transmit header.
+
+.. _ug_nrf70_developing_raw_packet_transmit_operation:
+
+Raw 802.11 packet transmit operation
+************************************
+
+The raw packet data has to be encompassed in an 802.11 packet and prepended with the raw transmit header before it is transmitted via raw socket to the nRF70 Wi-Fi device driver for transmission. The packet is forwarded to the nRF70 Series device which transmits the packet over the air. All 802.11 MAC layer rules for the 802.11 packet (e.g. link-layer acknowledgements, retransmissions etc) are handled by the nRF70 Series device. The packet structure and raw packet operation is depicted below:
+
+.. figure:: images/nrf7000_packet_Injection_tx_Flow.svg
+   :alt: Raw packet transmit packet structure and raw packet operation flow
+
+   Raw packet transmit packet structure and raw packet operation flow
+
+.. _ug_nrf70_developing_channel_operation:
+
+Channel operation
+*****************
+
+Channel configuration for raw packet transmission during station idle mode is necessary. Though there is a fallback channel on which packets are transmitted when the raw packet channel is not configured, It is desirable to configure an explicit channel of operation for raw packets when station is in disconnected idle state. Raw packet transmissions will follow the below channel operation for raw packet transmission:
```

Comment:
I do not think you have defined what you mean by "Station idle mode", so this could be a bit confusing, at least to me.
Do you simply mean that the user needs to configure the channel for raw packet transmission when the nRF70 Series device operates in Station mode but is not connected to an AP?

Reply:
Also below you refer to "disconnected idle state"
Unless you this means something else, I would rather see us using a consistent terminology. So here it would be "non-connected" Station mode

Reply:
will re-word this. Need to check how to

Reply:
OK great - will wait for the new version

Reply:
I just re-worded to *non-connected station* which are now aligned with the rest of the document. Kindly check